vor-android/mobile/src/main/java/com/futurice/hereandnow/pojo/Toilet.kt

```diff
@@ -1,19 +0,0 @@
-package com.futurice.hereandnow.pojo
-
-import android.content.SharedPreferences
-
-/**
- * Created by Luís Ramalho on 28/12/15.
- * <luis.ramalho@futurice.com>
- */
-class Toilet {
```

Comment:
I feel that we can leave it and slowly try to bring Kotlin into to the app, as I become more confident with Kotlin I plan to use more thoroughly. It would actually be one of my plus points to work on this project afterwards :pray: